sh/tests/grades_test.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+IFS='
+'
+script_dirS=$(cd -P "$(dirname "$BASH_SOURCE")" &>/dev/null && pwd)
+
+challenge() {
+    args=$1
+    input=$2
```

Comment:
taking `args` and `input` like that doesn't allow you to test behavior when more or less than one argument are passed to `grades.sh`.

You could use this kind of notation to achieve that goal:
```bash
args=${@:1:$#-1}
input=${@: -1}
```

This is a bit of a bashinsm, and could have some portability draw-backs if used with different terminals, but here it should be quite safe to assume we will always run that script using bash since we have control over the execution environment.

If you find other ways to implement this please feel free to do so, as soon as it allows to test variadic arguments passed to the script.

Reply:
done, it was not working before so i pushed with the other fix, it turns out we need the double quotes 
```
    input="${@: -1}"
    args=${@:1:$#-1}
```